src/objective-c/GRPCClient/GRPCTypes.h

```diff
@@ -131,13 +131,6 @@ typedef NS_ENUM(NSUInteger, GRPCCallSafety) {
    * state.
    */
   GRPCCallSafetyDefault = 0,
```

Comment:
Do we plan to introduce new request features in the near term ? if not, this removal basically renders the entire GRPCCallSafety enum obsolete. I can create a follow up TODO to remove the enum if that's the case. Thanks

Reply:
I think that would be reasonable to do!

I'm not aware of any upcoming features that would want a similar API, @markdroth?

Reply:
No immediate plans, no.  But in the long run, I would not be surprised if we wind up re-adding GET support.